Jenkinsfile

```diff
@@ -392,8 +389,64 @@ pipeline {
               }
               build(job: 'apm-server/apm-hey-test-benchmark', propagate: true, wait: true,
                     parameters: [string(name: 'GO_VERSION', value: '1.12.1'),
-                                string(name: 'STACK_VERSION', value: "${env.GIT_BASE_COMMIT}"),
-                                string(name: 'APM_DOCKER_IMAGE', value: "${env.DOCKER_IMAGE}")])
+                                 string(name: 'STACK_VERSION', value: "${env.GIT_BASE_COMMIT}"),
+                                 string(name: 'APM_DOCKER_IMAGE', value: "${env.DOCKER_IMAGE}")])
+            }
+          }
+        }
+        stage('Package') {
+          agent { label 'linux && immutable' }
+          options { skipDefaultCheckout() }
+          environment {
+            PATH = "${env.PATH}:${env.WORKSPACE}/bin"
+            HOME = "${env.WORKSPACE}"
+            GOPATH = "${env.WORKSPACE}"
+            SNAPSHOT = "true"
+          }
+          when {
+            beforeAgent true
+            allOf {
+              expression { return params.release_ci }
+              expression { return env.ONLY_DOCS == "false" }
+            }
+          }
+          stages {
+            stage('Package') {
+              steps {
+                withGithubNotify(context: 'Package') {
+                  deleteDir()
+                  unstash 'source'
+                  golang(){
+                    dir("${BASE_DIR}"){
+                      sh(label: 'Build packages', script: './script/jenkins/package.sh')
+                      sh(label: 'Test packages install', script: './script/jenkins/test-install-packages.sh')
+                      dockerLogin(secret: env.DOCKER_SECRET, registry: env.DOCKER_REGISTRY)
+                      sh(label: 'Package & Push', script: "./script/jenkins/package-docker-snapshot.sh ${env.GIT_BASE_COMMIT} ${env.DOCKER_IMAGE}")
```

Comment:
I think docker is not available on the golang docker image

Reply:
It should be there as https://github.com/elastic/apm-server/pull/3488 was created for that and it did work unless it has been changed recently